### PR TITLE
Prepare 0.11.1 release

### PR DESCRIPTION
**Pull request by tunetheweb**
There have been numerous changes to the core `fix` code so looking to do a 0.11.1 release.

@R7L208 @barrywhart @OTooleMichael @rpr-ableton you have open PRs that look like they could make this release so gentle nudge to get them closed out in next day or two if you want them in.

### CODE REVIEW

File: CHANGELOG.md
Comment (barrywhart) [inline CHANGELOG.md:21]:
```suggestion
* Numerous grammar changes
```

**Review by barrywhart — COMMENTED** [review]
Looks good so far!

**Comment by R7L208** [comment]
My mac updated last night to Monterey 12.3 which removes python2.7. My local git instance is broken as a result so can't push the latest changes to my outstanding PR. I'm working through it but wouldn't hold up the release for me.

@tunetheweb - other than the feedback you gave, there's one other minor naming change that I want to make before merge.

**Comment by tunetheweb** [comment]
Think it's pretty much there @R7L208 so let me know in that PR what you want changed and I can finish it off for you.

**Comment by tunetheweb** [comment]
@OTooleMichael / @barrywhart all open PRs that I expected to merge are done apart from L003 refactor in #2857 . Should we hold up for that, or is it still a bit off? If it is, then tempted to release as is.

**Comment by barrywhart** [comment]
@tunetheweb: Releasing now seems good to me. The L003 PR is pretty good overall, but we just started review late today. It's a solid improvement to L003, but doesn't seem high priority enough to wait for.

File: CHANGELOG.md
Comment (barrywhart) [inline CHANGELOG.md:26]:
```suggestion
* Add Marker to list of organizations using SQLFluff in the wild  [#2874](https://github.com/sqlfluff/sqlfluff/pull/2874) [@kdw2126](https://github.com/kdw2126)
```

**Review by barrywhart — APPROVED** [review]

File: CHANGELOG.md
Comment (tunetheweb) [reply CHANGELOG.md:26]: Nope, apparently it's Markerr: https://www.markerr.com

Comment (barrywhart) [reply CHANGELOG.md:26]: Oh wow! It _looked_ like a typo, heh.

**Pull request merged**
